app/src/main/res/xml/camera_widget_info.xml

```diff
@@ -3,8 +3,8 @@
     android:configure="io.homeassistant.companion.android.widgets.camera.CameraWidgetConfigureActivity"
     android:initialKeyguardLayout="@layout/widget_camera"
     android:initialLayout="@layout/widget_camera"
-    android:minWidth="250dp"
-    android:minHeight="80dp"
+    android:minWidth="40dp"
```

Comment:
Should `android:targetCellWidth="2"`, `android:targetCellHeight="2"` be added so the widget keeps the current size in the launcher widget gallery? Allowing 1x1 is nice but by default it is quite small.

Reply:
yes thats a great suggestion, thank you :)